<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Haru Wallet</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(42, 42, 42)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="walletName">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>240</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HYWenHei-85W</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Haru Wallet</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(167, 167, 167)</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="credits">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>550</y>
      <width>210</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HYWenHei</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Отправить&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Haru Software, 2023</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="keyInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HYWenHei</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Выберите аккаунт:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="walletAbout">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>70</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HYWenHei</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Менеджер аккаунтов</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="keyEnter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HYWenHei</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 10px;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addAccount">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HYWenHei</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { border-radius: 10px; 
 min-height: 20px;
 min-width: 20px; 
 background-color: rgb(0, 170, 0); }
QPushButton:hover {
	background-color: rgb(0, 255, 0);
  padding: 10px; /* отступ внутри кнопки */
  transition: background-color 0.5s, color 0.5s, border-radius 0.5s, padding 0.5s; /* параметры анимации */
}</string>
    </property>
    <property name="text">
     <string>Добавить аккаунт</string>
    </property>
   </widget>
   <widget class="QLabel" name="receiverAddressLabel_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HYWenHei</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Отправить&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;или&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="makeNewBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HYWenHei</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { border-radius: 10px; 
 min-height: 20px;
 min-width: 20px; 
 background-color: rgb(170, 0, 0); }
QPushButton:hover {
	background-color: rgb(170, 0, 0);
  padding: 10px; /* отступ внутри кнопки */
  transition: background-color 0.5s, color 0.5s, border-radius 0.5s, padding 0.5s; /* параметры анимации */
}</string>
    </property>
    <property name="text">
     <string>Создать новый аккаунт</string>
    </property>
   </widget>
   <widget class="QLabel" name="addressBalance_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HYWenHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Отправить&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(166, 166, 166)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите ключ (если необходимо): &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="delAccount">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HYWenHei</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { border-radius: 10px; 
 min-height: 20px;
 min-width: 20px; 
 background-color: rgb(0, 170, 0); }
QPushButton:hover {
	background-color: rgb(0, 255, 0);
  padding: 10px; /* отступ внутри кнопки */
  transition: background-color 0.5s, color 0.5s, border-radius 0.5s, padding 0.5s; /* параметры анимации */
}</string>
    </property>
    <property name="text">
     <string>Удалить аккаунт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continueWith">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HYWenHei</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { border-radius: 10px; 
 min-height: 20px;
 min-width: 20px; 
 background-color: rgb(0, 170, 0); }
QPushButton:hover {
	background-color: rgb(0, 255, 0);
  padding: 10px; /* отступ внутри кнопки */
  transition: background-color 0.5s, color 0.5s, border-radius 0.5s, padding 0.5s; /* параметры анимации */
}</string>
    </property>
    <property name="text">
     <string>Продолжить с выделенным</string>
    </property>
   </widget>
   <widget class="QListWidget" name="accountListWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HYWenHei</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="button_icons.qrc"/>
 </resources>
 <connections/>
</ui>
